update.sh

```diff
@@ -31,16 +31,22 @@ sudo mv rhino-config /usr/bin
 rm -rf ~/rhino-config
 
 # If the user has selected the option to install the mainline kernel, install it onto the system.
-if [[ -f "$HOME/.rhino/config/mainline" ]]; then
-  if [[ ! -f "$HOME/.rhino/config/5-17-9" ]]; then
+if [[ -f "$HOME/.rhino/config/mainline" ]] && [[ ! -f "$HOME/.rhino/config/5-17-9" ]]; then
     cd ~/rhinoupdate/kernel/
+    wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.17.9/amd64/CHECKSUMS
```

Comment:
Will the checksums url have to be updated every time a new kernel is released?

Reply:
Obviously?

Reply:
Okay just confirming